mysql-test/std_data/keyring_vault_confs/keyring_vault.conf

```diff
@@ -0,0 +1,4 @@
+vault_url = https://127.0.0.1:8200
```

Comment:
I have moved the tests under plugin/keyring_vault, apart from std_data as we discussed on slack.